xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererGuiTexture.cpp

```diff
@@ -73,15 +70,10 @@ CRPRendererGuiTexture::CRPRendererGuiTexture(const CRenderSettings &renderSettin
 
 bool CRPRendererGuiTexture::Supports(RENDERFEATURE feature) const
 {
-  if (feature == RENDERFEATURE::STRETCH         ||
-      feature == RENDERFEATURE::ZOOM            ||
-      feature == RENDERFEATURE::PIXEL_RATIO     ||
-      feature == RENDERFEATURE::ROTATION)
-  {
-    return true;
-  }
-
-  return false;
+  return feature == RENDERFEATURE::STRETCH         ||
```

Comment:
Would a switch statement improve this function?

Reply:
I think it's OK to just return, feels more direct.